RLBotDiscordBot/bot.py

```diff
@@ -63,7 +63,7 @@ def __init__(self):
                             self.has_checked = True
 
         if message.author.bot:
-            if message.channel.id == 604049792284360864:
+            if message.channel.id == 352507627928027138:
                 self.has_reacted += 1
                 if self.has_reacted % 2 == 0:
                     reaction_list = ["👍","👀","🔥","👌","😄","😮","<:scratchcat:444921286703972352>","<:rank_quantum:592004043832950784>"]
```

Comment:
adding a command to edit the emoji list would be cool as well

Reply:
I'll do that tomorrow

Reply:
thanks!